OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -309,7 +309,7 @@ public void RefreshIcons()
 				var y = DisplayedIconCount / Columns;
 				var rect = new Rectangle(rb.X + x * (IconSize.X + IconMargin.X), rb.Y + y * (IconSize.Y + IconMargin.Y), IconSize.X, IconSize.Y);
 
-				var rsi = item.Traits.Get<RenderSpritesInfo>();
+				var rsi = item.Traits.WithInterface<RenderSpritesInfo>().First(t => t.IsTraitEnabled());
```

Comment:
Actors should never have more than one `RenderSprites` trait, so this is not necessary.